src/Jobs/RemoveFromSearch.php

```diff
@@ -38,7 +38,9 @@ public function __construct($models)
     public function handle()
     {
         if ($this->models->isNotEmpty()) {
-            $this->models->first()->searchableUsing()->delete($this->models);
+            $this->models->first()->searchableUsing()->delete(
+                RemoveableScoutCollection::make($this->models)
+            );
```

Comment:
We can't make this change. The `RemoveableScoutCollection` instance must be present in the constructor for the `getQueueableIds()` method to return the proper scout keys. This collection method will be called when the job is serialized and placed onto the queue:

https://github.com/laravel/scout/blob/3352fbc5c071cbd3d5392222363f06cf660ca003/src/Jobs/RemoveableScoutCollection.php#L15